packages/backend/src/yorkie/worksheet-settings.ts

```
@@ -0,0 +1,76 @@
+  return {
+    rows: assertNonNegInt(b.rows ?? 0, 'rows'),
+    cols: assertNonNegInt(b.cols ?? 0, 'cols'),
+  };
+}
+
+/** Validate a hidden body `{ rows: number[], columns: number[] }` (0-based indices). */
+export function parseHidden(body: unknown): {
+  rows: number[];
+  columns: number[];
+} {
+  const b = assertObject(body, 'hidden body must be an object { rows, columns }');
```

ESLint

[error] Replace `body,·'hidden·body·must·be·an·object·{·rows,·columns·}'` with `⏎····body,⏎····'hidden·body·must·be·an·object·{·rows,·columns·}',⏎··`

(prettier/prettier)